src/Core/src/Fonts/FontManager.Windows.cs

```diff
@@ -27,6 +28,9 @@ public class FontManager : IFontManager
 		readonly IFontRegistrar _fontRegistrar;
 		readonly IServiceProvider? _serviceProvider;
 
+		FontFamily? _defaultFontFamily;
```

Comment:
I would suggest a comment about why it is cached so that a future programmer knows not to mess with it.